services/QuillLMS/app/assets/stylesheets/pages/my_lessons.scss

```diff
@@ -2,58 +2,30 @@
   section:last-of-type {
     padding-bottom: 5px;
   }
-  .my-lessons-header {
-    margin-top: 30px;
-    margin-bottom: 30px;
-    h1 {
-      font-weight: 600;
-      font-size: 30px;
-      margin-bottom: 10px;
-    }
-    p {
-      margin: 7px 0px;
-      font-size: 16px;
-    }
-    span {
-      font-weight: 600;
+  .top-section {
+    display: flex;
+    margin: 32px 0px;
+    .my-lessons-header {
+      h1 {
+        @include display-xl;
+        margin-bottom: 10px;
+      }
+      p {
+        margin: 7px 0px;
+        @include text-s;
+      }
+      span {
+        font-weight: 600;
+      }
+      a {
+        color: #027360;
```

Comment:
Maybe we could substitute this for a color variable?

Reply:
good catch! with the refactoring, I was in a bit of autopilot for some parts of copy and pasting 😅